draft-ietf-quic-multipath.md

```diff
@@ -592,25 +592,23 @@ as "no packet received on any path for the duration of the idle timeout".
 When only one path is available, servers MUST follow the specifications
 in {{QUIC-TRANSPORT}}.
 
-When more than one path is available, hosts shall monitor the arrival of
-packets and acknowledgements for packets sent over each
-path. Hosts MAY consider closing a path if for at least the period of the
-idle timeout as specified in {{Section 10.1. of QUIC-TRANSPORT}}
-(a) no packet was received or (b) no packet sent over this path was acknowledged.
-Endpoints that desire to close a path because of the idle timer rule
-MUST do so explicitly by sending a PATH_ABANDON frame on another path,
-as defined in {{path-close}}.
-
-To avoid idle timeout of a path, endpoints
+This document does not specify any specific per-path timeouts. An endpoint
+can decide to close a path at any time, whether the path is in active
+use or not, by sending a PATH_ABANDON frame. It is not required
+to send a PATH_ABANDON frame at any specific point in time.
+For example, an endpoint may wait until it will anyway send another frame.
```

Comment:
> It is not required to send a PATH_ABANDON frame at any specific point in time.

Do you mean "An endpoint does not need to send a PATH_ABANDON frame immediately or at any particular moment"?

Reply:
yes, one endpoint decides to close a path and then need to send an abandon frame, however, there is nothing in this spec that says it has to take that decision as an specific point in time, e.g. as react to a certain event. For example even if you receive a packet on a new path that you don't want to open, it doesn't necessarily mean that you have to close it immediately, because opening a path (by receiving) a packet is an independent event from closing the path, meaning as you as you have received the path it is effetely on open path and similarly as you can decide to close any path at any point of time, you simply also decide to close that path short after opening it. Sorry for using these many works but I hope that makes my mental model more clear....?